package spob;

message AckTree {
  required uint32 primary = 1;
  required uint64 count = 2;
  required uint64 last_mid = 3;
}